e2e/process-services-cloud/start-task-custom-app-cloud.e2e.ts

```diff
@@ -15,22 +15,23 @@
  * limitations under the License.
  */
 
-import { LoginSSOPage } from '../pages/adf/loginSSOPage';
+import { LoginSSOPage } from '../../lib/testing';
 import { SettingsPage } from '../pages/adf/settingsPage';
 import { AppListCloudComponent } from '../pages/adf/process-cloud/appListCloudComponent';
 import TestConfig = require('../test.config');
 import { NavigationBarPage } from '../pages/adf/navigationBarPage';
 import { TasksCloudDemoPage } from '../pages/adf/demo-shell/process-services/tasksCloudDemoPage';
-import { StartTasksCloudComponent } from '../pages/adf/process-cloud/startTasksCloudComponent';
+import { StartTasksCloudComponent } from '../../lib/testing';
```

Comment:
Can we have as default the page as part of the class name instead of component?
StartTasksCloudPage